html-report-generator/generate_usage_report.py

```diff
@@ -1,33 +1,30 @@
 #!/usr/bin/env python
 
 import datetime
-import sys
 import itertools
 from os.path import dirname, abspath, basename, join as pjoin
 
+# Use Agg non-interactive backend to prevent X forwarding error
+import matplotlib
+matplotlib.use('Agg')
```

Comment:
Why is this coming up in diff? I had already committed it to the PR that you merged.

Reply:
Moved it above the other imports, as we want it to initialize matplotlib with Agg before it imports pyplot in the other files.